openedx/features/course_experience/utils.py

```diff
@@ -115,6 +115,7 @@ def recurse_mark_auth_denial(block):
             'completion',
             'complete',
             'resume_block',
+            'hide_from_toc',
```

Comment:
why is this change necessary?

Reply:
If the field is not added to the `requested_fields` list, the field is returned `null`